### ISSUE TITLE
Replace `web_time` with `jiff`

### ISSUE THREAD

**Issue by tisonkun** [fast/fastant#2]
@burntsushi I found `jiff` has wasm support but I'm yet to investigate if `jiff` can replace [`web_time`](https://crates.io/crates/web_time). I'll try to figure it out in day, but first let you know this is a possible collaboration point, and if you have time to provide some inputs, it can be very helpful.

**Comment by BurntSushi** [comment]
`Zoned::now()` and `Timestamp::now()` and `TimeZone::system()` will all "just work" on `wasm32-unknown-unknown` targets when the `js` crate feature is enabled. You can read more about WASM support here: https://docs.rs/jiff/latest/jiff/_documentation/platform/index.html#wasm

`web_time` also seems to provide monotonic time, i.e., a `std::time::Instant` replacement. Jiff's doesn't deal with monotonic time. So if you're using monotonic time, you might still need to use `web_time` for that.

**Comment by tisonkun** [comment]
> Jiff's doesn't deal with monotonic time

@BurntSushi out of curiosity, could you tell me what should be done to provide monotonic time and why Jiff decides not to do that?

**Comment by BurntSushi** [comment]
They are two different use cases. Monotonic time is typically used for measuring "how long" something took in the context of a program. But crucially, you can't convert monotonic time to something like a human notion of time. There's no way to go from a `std::time::Instant` to something like a Gregorian calendar datetime for example.

Jiff doesn't deal with monotonic time because monotonic time is much more limited in scope and is already adequately covered by [`std::time::Instant`](https://doc.rust-lang.org/std/time/struct.Instant.html). I might suggest reading those docs too, because it explains things in a bit more detail. One practical point here is that monotonic time provides a guarantee about the values it produces, where as `std::time::SystemTime` does not. Like, you, as a user of your system, can change your system's time. And that can in turn impact programs reading your system time. Which is good. But you, as a user, can do nothing to impact monotonic time. Which is also good.

Jiff is all about dealing with civil datetimes and instants in time that humans use to coordinate scheduling. This is a much more complex problem than monotonic time. Monotonic time is really just about exposing support from the underlying platform and being able to measure the duration between two points. But it stops there, because there is no correspondence between a monotonic time and a real world datetime.

TL;DR - monotonic time and calendar datetime are two very different things.

**Comment by tisonkun** [comment]
Thanks a lot for your explanation! It's very clear.

Closed as won't do.
